docker/rootfs/etc/services.d/go2rtc/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ -f "/config/frigate-go2rtc.yaml" ]; then
```

Comment:
```suggestion
if [[ -f "/config/frigate-go2rtc.yaml" ]]; then
```

double brackets are preferred in bash according to [GSSG](https://google.github.io/styleguide/shellguide.html).